_compdemos/Mountain-CyberDuck.md

```diff
@@ -43,49 +43,47 @@ If required, ask the Helpdesk or your divisional IT support to be added to the s
 - To get a license file, download the `fredhutch50.mountainducklicense` from [here](https://teams.fhcrc.org/sites/citwiki/SciComp/Documents/fredhutch50.mountainducklicense) and double click the file or drag it to the Mountain Duck application icon to register.
 - Continue with [Configuring Mountain Duck](#Configuring-Mountain-Duck) below.
 
 ### Configuring Mountain Duck
 
 - Start Mountain Duck (if not already started) and click the Tray icon (see above)
-- Select "New Bookmark" and then profile "OpenStack Swift Auth v2.0 (SwiftStack HTTPS)" from the pull down menu
-- Enter the following settings: (lastname_f is the hutch investigator ID, eg. gilliland_g or groudine_m, username is the hutchnet id)
+- Select "New Bookmark" and then profile "Amazon S3" from the pull down menu
+- Refer to the encrypted email you received when your lab's AWS account was [set up](/scicomputing/access_credentials/#amazon-web-services-aws)
+- Enter the following settings: Access Key ID, Secret Access Key. Change Nickname to something memorable, such as "My Lab's S3 Buckets".
 
->Nickname: Eco_lastname_f (or whatever you like, should be unique per PI)
-Server: tin.fhcrc.org
-Tenant Name:User Name: AUTH_Swift_lastname_f:username  
-(case sensitive: this is the Tenant name of the PI and your user name separated by a colon, e.g. AUTH_Swift_gilliland_g:markg or AUTH_Swift_groudine_m:gary etc. )
-Path: /auth/v​2.0 (optional, not required in Swiftstack profile)
 
-![](/assets/store_objectstore_swift/2018-09-25-11-26-44.png)
+![](/assets/mountain_cyber_duck/2022-08-15-10-33-25.png)
 
-After you are done, click "Connect" to save the settings, you will be prompted for your password. After entering your password wait a few seconds. Your drive should open or you should see it in Windows Explorer.
-If you are having problems please check How to troubleshoot MountainDuck and CyberDuck
+After you are done, click "Connect" to save the settings, you will be prompted for your password. After entering your password wait a few seconds. Your drive should open or you should see it in Windows Explorer or MacOS Finder.
+If you are having problems please check How to troubleshoot MountainDuck and CyberDuck.
 
 
 ## Cyberduck
 
 ### Installing Cyberduck
 
-If required ask the Helpdesk or your divisional IT support to be added to the security group lastname_f_grp of your PI (e.g. gilliland_g_grp). Make sure you CC your PI or Manager to let them know that you are requesting access.
-If you are installing Cyberduck yourself (e.g on a Mac) you need to download the Swiftstack profile from [here](http://files.swiftstack.com/OpenStack%20Swift%20Auth%20v2.0%20(SwiftStack%20HTTPS).cyberduckprofile) and open it from your downloaded files. 
+If required ask the Helpdesk or your divisional IT support to be added to the security group lastname_f_grp of your PI (e.g. lynch_t_grp). Make sure you CC your PI or Manager to let them know that you are requesting access.
```

Comment:
I think we can drop this bit about getting added to the group.  It was necessary for accessing economy local, but with cloud it's a completely different auth mechanism.